controller/array_action/array_mediator_ds8k.py

```diff
@@ -660,19 +655,16 @@ def get_host_by_host_identifiers(self, initiators):
             logger.debug("can not found host by initiators: {0} ".format(initiators))
             raise array_errors.HostNotFoundError(initiators)
 
-    def validate_supported_capabilities(self, capabilities):
-        logger.debug("Validating capabilities: {0}".format(capabilities))
+    def validate_supported_space_efficiency(self, space_efficiency):
+        logger.debug("validate_supported_space_efficiency for space efficiency : {0}".format(space_efficiency))
 
-        # Currently, we only support one capability "SpaceEfficiency"
-        # The value should be: "thin"
-        if (capabilities and capabilities.get(
-                config.CAPABILITIES_SPACEEFFICIENCY).lower() not in
-                [config.CAPABILITY_THIN, ]):
-            logger.error("capabilities is not supported.")
-            raise array_errors.StorageClassCapabilityNotSupported(
-                capabilities)
+        if (space_efficiency and space_efficiency.lower() not in
+                [config.SPACE_EFFICIENCY_THIN, ]):
```

Comment:
shouldn't we also support explicit `standard`?

Reply:
we can add (I believe you ment 'none', I don't know what is 'standard')

Reply:
right, sorry, was looking at an [old version](https://www.ibm.com/support/knowledgecenter/SSRQ8T_1.3.0/csi_ug_config_create_storageclasses.html) of the documentation

Reply:
can we resolve?

Reply:
done